pkg/util/stripsecrets.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"strings"
```

Comment:
could protosanitizer work here? https://github.com/kubernetes-csi/csi-lib-utils/blob/master/protosanitizer/protosanitizer.go

Reply:
I had a quick glimpse of the code before and it didn't seem like it could be of any use in this case since it expects JSON as input. Do you think it be somehow reused here with less code than it currently needs?